tardis/tardis_portal/static/js/facility_view.js

```diff
@@ -153,11 +153,13 @@
 
     // Group facilities data by user
     function groupByUser(data) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['groupByUser' is defined but never used.](https://www.codacy.com/app/mytardis/mytardis/file/600512116/issues/source?bid=2125231&fileBranchId=2278801#l155)